quickjs/src/jniTest/kotlin/app/cash/quickjs/CallBridgesTest.kt

```diff
@@ -110,7 +105,7 @@ internal class CallBridgesTest {
   }
 
   @Test
-  fun suspendingRequestAndResponse() {
+  fun suspendingRequestAndResponse() = dispatcher.runBlockingTest {
```

Comment:
We've banned runBlockingTest internally because it has some really, really bad behavior. Like it's 100% broken for some coroutine shapes. Normally I recommend `runBlocking(dispatcher)` in place.

If it works for now we can leave it. On the first problem I am going to replace every single usage. It's a bad API because it's just a timebomb waiting to happen.

Reply:
Good call. Switched.